File: gofCsharp/gofCsharp/gofCsharp/Printer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace gofCsharp
{
    public class Printer : IPrinter
    {

        private char TrueCell;
        private char FalseCell;
        public Printer()
        {
            TrueCell = '*';
            FalseCell = '.';
        }
        public void Print(Board pop)
        {
            var matrix = pop.Population;

            for (int i = 0; i < pop.Width; i++)
            {
                for (int j = 0; j < pop.Height; j++)
                {
                    var state = matrix[i, j];
                    Console.Write($"{(state ? TrueCell : FalseCell)}");
                }
                Console.WriteLine();
            }

        }
    }
}

File: gofCpp/gofCpp/gofCpp/Printer.cpp
#include "Printer.h"
#include "iostream"

using namespace std;

Printer::Printer(char trueCell, char falseCell)
{
	this->falseCell = falseCell;
	this->trueCell = trueCell;
}

void Printer::print(Board * b)
{
	for (int i = 0; i < b->width; i++)
	{
		for (int i = 0; i < b->height; i++)
		{
			cout << '*';

		}
		cout << endl;
	}
}
